lib/user.rb

```diff
@@ -0,0 +1,31 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/joke'
+require 'pry'
+
+class User
+  attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @joke = joke
+    @jokes << @joke
+  end
+
+  def sal_tells(name, joke)
+    @name = name
+    @joke = joke
+    @jokes << @joke
+  end
```

Comment:
If we called this method `.tell` and called `@sal.tell(@ali, @joke_1)` how would we need to fix this code so Sal could teach Ali the joke?